specs/NonClientHitTestKind.md

```diff
@@ -0,0 +1,208 @@
+        CHECK_FAILURE(m_compositionController->GetHitTestResultAtPoint(&point, &result));
+
+        if (result == COREWEBVIEW2_HIT_TEST_RESULT_HTCAPTION)
+        {
+            return HTCAPTION;
+        }
+
+        break;
+    ....
+}
+....
+}
+```
+#### C#/WinRT
+```c#
+protected override void WndProc(ref Message m)
+{
+....
+
+if (m.Msg == WM_NCHITTEST)
+{
+    Point point = new Point(GET_X_LPARAM(m.lParam), GET_Y_LPARAM(m.lParam));
+    if (m_compositionController.GetHitTestResultAtPoint(point) == 
+        CoreWebView2HitTestResult.Htcaption) 
+    {
+        m.Result = (IntPtr)HTCAPTION;
+    }
+}
+// The following code forwards mouse messages to the WebView2 - Start
+else if (m.msg >= WM_MOUSEFIRST && m.msg <= WM_MOUSELAST 
+        || m.msg == WM_NCRBUTTONDOWN || m.msg == WM_NCRBUTTONUP) 
+{
+  
+    Point point = new Point(GET_X_LPARAM(m.lParam), GET_Y_LPARAM(m.lParam));
+    // IsPointInWebview is an example of a function which your app can implement that 
+    // checks if the mouse message point lies inside the rect of the visual that hosts
+    // the WebView.
+    if (IsPointInWebView(point) || m.msg == WM_MOUSELEAVE) 
+    {
+        
+        // forward mouse messages to WebView2
+        m_compositionController.SendMouseInput(
+            (CoreWebView2MouseEventKind)message,
+            (CoreWebView2MouseEventVirtualKeys)GET_KEYSTATE_WPARAM(wParam),
+            mouseData,
+            point);
+
+        m.Result = 0
+        return;
+    }
+        
+}
+// The following code forwards mouse messages to the WebView2 - Start
+    ....
+}
+```
+# API Notes
+## Marking regions In Your App
+Draggable regions in WebView2 are HTML elements marked with the CSS style
+`-webkit-app-region: drag`. WebView2 will treat any element with this style as a 
+drag handle for the window.
+
+See Documentation [here](https://learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/window-controls-overlay#make-regions-of-your-app-drag-handlers-for-the-window)
+
+# API Details
+## Win32 C++
+```cpp
+/// This enum contains values representing possible regions a given
+/// point lies within
+typedef enum COREWEBVIEW2_HIT_TEST_RESULT {
+    /// A hit test region in the WebView2 which has the CSS style 
+    /// `-webkit-app-region: drag` set. Web content should use this CSS 
+    /// style to identify regions that should be treated like the app 
+    /// window's title bar. This has the same value as the Win32 HTCAPTION 
+    /// constant. 
+    COREWEBVIEW2_HIT_TEST_RESULT_CAPTION = 2,
+    /// A hit test region in the WebView2 which does not have the CSS style 
+    /// `-webkit-app-region: drag` set. This is normal web content that should not be 
+    /// considered part of the app window's title bar. This has the same value 
+    /// as the Win32 HTCLIENT constant.
+    COREWEBVIEW2_HIT_TEST_RESULT_CLIENT = 1,
+    /// A hit test region out of bounds of the WebView2.
+    COREWEBVIEW2_HIT_TEST_RESULT_NOWHERE = 0
```

Comment:
Probably should be NO_WHERE so that they show up as NoWhere in .NET & WinRT. Would also be good to add a note to docs that this has the same value as the Win32 HTNOWHERE constant.